coriolis/osmorphing/oracle.py

```diff
@@ -22,6 +22,14 @@ def _run_dracut(self):
         self._run_dracut_base('kernel')
         self._run_dracut_base('kernel-uek')
 
+    def _find_oracle_repos(self, repos_to_enable):
```

Comment:
This appears to generally apply to any RHEL-based system so please move it to the base RHEL tools class

Reply:
how to  install on linux  server？  so  like  centos  Ubuntu  ，&& suse